views/index.ejs

```diff
@@ -155,6 +155,12 @@
               "id": "flush-headingTwo",
               "id2": "flush-collapseTwo",
             },
+            {
+              "question": "How do I create a new account (wallet)?",
+              "answer": `Follow the instructions here: https://dev.poktroll.com/operate/user_guide/create-new-wallet.`,
```

Comment:
```suggestion
              "answer": `Follow the instructions here: <a href='https://dev.poktroll.com/operate/user_guide/create-new-wallet' target='_blank'>https://dev.poktroll.com/operate/user_guide/create-new-wallet</a>.`,
```

This should be a link, otherwise LGTM.